studio/src/editors/merch-card-editor.js

```diff
@@ -181,33 +302,271 @@ class MerchCardEditor extends LitElement {
                 value="${this.fragment.tags.map((tag) => tag.id).join(',')}"
                 @change=${this.#handeTagsChange}
             ></aem-tag-picker-field>
-        `;
+            <p>Fragment details (not shown on the card)</p>
```

Comment:
This is specific to merch cards but common to any fragment, hence it was put in `editor-panel.js`.
Please move any form logic that is not specific to merch-card, back to editor-panel.js